testing/test-helper.js

```diff
@@ -105,6 +105,52 @@ export function assertScreenReaderElement(element) {
   expect(computedStyle.getPropertyValue('visibility')).to.equal('visible');
 }
 
+/**
+ * Expects the function call to generate console error messages.
+ *
+ * The expectedErrorMessages parameter is a list of strings or RegExp objects
+ * to match against the output of each call to console.error.
+ *
+ * e.g.:
+ * expectConsoleError(() => {
+ *   console.error('This is the first error');
+ *   console.warn('Warns and logs are not caught');
+ *   console.error('This is the second error');
+ * }, ['This is the first error', /second/]);
+ *
+ * @param {!Function} func
+ * @param {!Array<string, !RegExp>} expectedErrorMessages
+ */
+export function expectConsoleError(func, expectedErrorMessages) {
```

Comment:
Grammar nit: Since we're expecting one or more errors, rename this to `expectConsoleErrors`.

Reply:
Done